jax/_src/numpy/lax_numpy.py

```diff
@@ -3847,6 +3847,36 @@ def diagflat(v, k=0):
   res = res.reshape(adj_length,adj_length)
   return res
 
+@_wraps(np.poly)
+def poly(seq_of_zeros):
+  seq_of_zeros = atleast_1d(seq_of_zeros)
+
+  dt = seq_of_zeros.dtype
+  seq_of_zeros = seq_of_zeros.astype(np.mintypecode(dt.char))
+
+  sh = seq_of_zeros.shape
+  if len(sh) == 2 and sh[0] == sh[1] != 0:
+    # import at runtime to avoid circular import
+    from . import linalg
+    seq_of_zeros = linalg.eigvals(seq_of_zeros)
+
+  if len(seq_of_zeros.shape) != 1:
```

Comment:
`if seq_of_zeros.ndim != 1`

Reply:
My changes in https://github.com/google/jax/pull/6777/commits/30633e3839cef7c7d29a72a90d1aeee7b8d9c701 implement this suggestion.